pkg/apis/core/types.go

```diff
@@ -5494,6 +5506,15 @@ type PodLogOptions struct {
 	// the actual log data coming from the real kubelet).
 	// +optional
 	InsecureSkipTLSVerifyBackend bool
+
+	// Return the specific log stream. Defaults to "All".
```

Comment:
please note explicitly here that any value except all would not work with the `TailLines` specified

Reply:
Also update the description of TailLines with the same note